_developer-files/connect/api/objects/form-properties/sources/saas/google-sheets-object.md

```diff
@@ -0,0 +1,54 @@
+---
+# -------------------------- #
+#     USING THIS TEMPLATE    #
+# -------------------------- #
+
+## NEED HELP USING THIS TEMPLATE? SEE:
+## https://docs-about-stitch-docs.netlify.com/reference/connect-templates/destination-form-property/
+## FOR INSTRUCTIONS & REFERENCE INFO
+
+
+# -------------------------- #
+#        CONTENT TYPE        #
+# -------------------------- #
+
+product-type: "connect"
+content-type: "api-form"
+form-type: "source"
+key: "source-form-properties-google-sheets-object"
+
+
+# -------------------------- #
+#        OBJECT INFO         #
+# -------------------------- #
+
+title: "Google Sheets Source Form Property"
+api-type: "platform.google-sheets"
+display-name: "Google Sheets"
+
+source-type: "saas"
+docs-name: "google-sheets" # This should be whatever integration.name is. Ex: LinkedIn Ads is linkedin-ads
+
+property-description: ""
+## Used to create a description for the object that doesn't adhere to the standard in _developers/connect/api/documentation/api-form-properties.html
+## See the Heap object for an example
+
+
+# -------------------------- #
+#      OBJECT ATTRIBUTES     #
+# -------------------------- #
+
+uses-start-date: true
+
+# Only source-specific attributes need to be listed here.
+# The following attributes are considered common,
+# and therefore don't need to be listed:
+# anchor_time, cron_expression, frequency_in_minutes, image_version, start_date 
+
+object-attributes:
+  - name: "spreadsheet_id"
+    type: "string"
+    required: true
+    description: "The unique identifier of your spreadsheet"
```

Comment:
It's also a good idea to link back to the setup docs for this, since locating the spreadsheet ID isn't totally straightforward. You can add something like `"Refer to the [{{ form-property.display-name }} documentation](link) for instructions on locating this info.` to the description here.